src/components/UserDetails/UserDatails.jsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import "./UserDetails.scss";
+import facebook from "./../../assets/images/icons/facebook.png";
+import twitter from "./../../assets/images/icons/twitter.png";
+import instagram from "./../../assets/images/icons/instagram.png";
+import youtube from "./../../assets/images/icons/youtube.png";
+
+const UserDatails = () => {
+  return (
+    <>
+      <section className="userDetails">
+        <div className="userDetails__identity">
+          <img
+            className="userDetails__identity__image"
```

Comment:
```suggestion
            className="userDetails__identity_image"
```
please follow [this](https://en.bem.info/methodology/naming-convention/#naming-rules) convention all over the project

Reply:
done